provisioner/components/k8s/cluster_setup.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+import ansible_runner
+import logging
+
+inventory_path = "/opt/halo/install_depot/cortx-halo/provisioner/components/k8s/setup_playbook/inventory.yaml"
```

Comment:
These variables should go to const.py at root directory.
HALO_BASE_PATH = "/opt/seagate/halo/"
PROVISIONER_PATH = HALO_BASE_PATH + 'install_depot/cortx-halo/provisioner/'
INVENTORY_PATH and so one.

Reply:
Moved variables to const.py